massa-execution/src/vm.rs

```diff
@@ -16,10 +16,18 @@ use std::sync::{Arc, Mutex};
 use tokio::sync::oneshot;
 use tracing::debug;
 
+/// Virtual Machine and step history system
 pub(crate) struct VM {
+    /// thread count
```

Comment:
Thread count is now in the static config, shouldn't it be available here, without the need to duplicate it here ? Or am I missing on something @yvan-sraka

Reply:
You right!

Reply:
And it can be removed from all functions ? Like Address::get_thread(thread_count) ?

Reply:
Can someone do this ? I'm curious to see how it's done in this particular case.  I am still confused about the new config system

Reply:
see #2197

Reply:
ok then it will be done as a followup

Reply:
I can make another PR this week if everybody is ok to resolve the #2197. For me we can merge this PR. @yvan-sraka @AureliaDolo @damip